array_api_strict/tests/test_elementwise_functions.py

```diff
@@ -19,8 +19,16 @@
 
 import pytest
 
+import array_api_strict
+
+
 def nargs(func):
```

Comment:
I modified this so that it works with decorated/wrapped functions as well. `len(getfullargspec(f).args)` returns zero for functions that are decorated with the array API version decorator. From what I can tell from the Python docs this is kind of on purpose/to preserve existing behaviour. `signature()` does the right thing for wrapped functions, but it needs slightly more explicit work to count the arguments.

I think the intention/rule is that `nargs()` counts the number of positional only arguments, which is basically the "number of arrays you need to pass to a elementwise function". I went with the very explicit way of counting the args partially as a way to make it easier for people from the future to understand what `nargs` is meant to do (even if it contains a bug and doesn't actually do what it is meant to do).